### PR TITLE
Update KL Divergence formula

### PR DESCRIPTION
**Pull request by goldmermaid**
Errors fix.

## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/gluon/loss.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ class KLDivLoss(Loss):
 
     .. math::
 
-        L = \sum_i {label}_i * \big[\log({label}_i) - {pred}_i\big]
+        L = \sum_i {label}_i * \big[\log({label}_i) - \log({pred}_i)\big]
```
Comment (sxjscience) [inline python/mxnet/gluon/loss.py]: It looks correct at first glance. However, I've checked the implementation again and it looks that the previous version is correct.

Comment (sxjscience) [reply python/mxnet/gluon/loss.py]: We are assuming the the preds are generated by the log-softmax activation.

**Comment by sxjscience** [comment]
@goldmermaid I think the 2nd change is valid, i.e., replacing pred with prob.
